### ISSUE TITLE
`topograph` or `top-down`

### ISSUE THREAD

**Issue by 4086606** [Templarian/MaterialDesign#6085]
I have:

- [x] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [x] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [x] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

A top-down variant of the `terrain` aka `mountain` icon. Design should look like a topograph.

## Examples

Should ideally be a topograph of the mountain in `terrain`.

**Comment by 4086606** [comment]
```xml
<svg version="1.1" viewBox="0 0 24 24">
    <path d="m2 4v2l2-2zm4 0-4 4v2l5-5 2 2 3-3h-2l-1 1-1-1zm8 0-5 5-2-2-5 5 5 5 2-2 5 5 8-8zm2 0 6 6v-2.828l-3.172-3.172zm4.828 0 1.172 1.172v-1.172zm-6.828 2 6 6-6 6-5-5-2 2-3-3 3-3 2 2zm0 2-4 4 4 4 4-4zm0 2 2 2-2 2-2-2zm-7 1-1 1 1 1 1-1zm-5 3v2l4 4h2l1-1 1 1h2l-3-3-2 2zm20 0-6 6h2.828l3.172-3.172zm-20 4v2h2zm20 .828-1.172 1.172h1.172z" />
</svg>
```

A first attempt at this icon..

**Comment by goyney** [comment]
That is very, very abstract which goes against Material guidelines.

**Comment by 4086606** [comment]
Yeah, realised. I think the general idea of a topograph is pretty abstract?
